_data/versions.yml

```diff
@@ -905,6 +905,41 @@ v3.3:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.3.0
 
 v3.2:
+- title: v3.2.8
+  components:
+    calico/node:
+      version: v3.2.8
+      url: https://github.com/projectcalico/node/releases/tag/v3.2.8
+    calicoctl:
+      version: v3.2.8
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.2.8
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.2.8/calicoctl
+    calico/cni:
+      version: v3.2.8
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.2.8
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.2.8/calico
```

Comment:
These `download_*_url`s are not correct for CNI. They both need `-amd64` appended or we need to add another asset. (The calicoctl one above is correct though.)

Reply:
It doesn't appear these URLs are referenced anywhere in v3.x pages or manifests actually. I'll try removing these `download_*_url` fields.

Reply:
These URLs are used in the integration guide https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/integration#installing-the-calico-cni-plugins

The requirements include amd64 so I think it's okay to not have the arch in the filename.

Reply:
There is no asset called `calico` in the release, there is only `calico-amd64`. Right now the wget in "Install the Calico plugins" will not work but it should.
So we either need to copy the `calico-amd64` and upload it as `calico` or update the url to include the `-amd64`.

Reply:
Ahh I see what you mean now. dee1aa3540df6b4c8a634fe82409dc44054b0f79

I did a check of the v3.1.x and v3.0.x URLs and those are fine with `calico`. For v3.2.x, the arch started being added to the assets in v3.2.3.